morepath/view.py

```diff
@@ -68,7 +68,7 @@ def __call__(self, app, obj, request):
             not request.app._permits(request.identity, obj, self.permission)):
             raise HTTPForbidden()
         if self.load is not None:
-            request.body_obj = body_obj = self.load(request)
+            body_obj = self.load(request)
             content = self.func(obj, request, body_obj)
```

Comment:
You can get rid of the local variable ``body_obj`` and inline the ``self.load(request)`` in the ``self.func`` call.